recipes/ifeature/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "ifeature" %}
+{% set version = "0.0.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: a96534f3a9777d72531efce19030c59311216d9ed8366856b6eed7cf5bb258dd
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+
+  run:
+    - python
+    - six ==1.15.0
+    - numpy >=1.10.4
+    - pandas >=0.18.1
+    - tqdm ==4.49.0
+    - lightgbm ==3.0.0
+    - jieba
+    - wrapt ==1.12.1
+
+
+test:
+  imports:
+    - __utils
+    - ifeature
+    - ifeature.eda
+    - ifeature.features_add
+    - ifeature.features_sub
+    - ifeature.features_transform
+    - ifeature.models
+    - ifeature.models.utils
+    - ifeature.utils
+
+about:
+  home: "https://github.com/Jie-Yuan/iFeature"
+  license: MIT
+  license_family: MIT
+  license_file: 
+  summary: "description"
```

Comment:
Could you add a description of your package here?

Reply:
fixed